### PR TITLE
fix: Update help center styles, fix routing issues

### PR DESCRIPTION
**Pull request by pranavrajs**
- Validate the last active portal against the list of portals in the account before navigation to avoid blank pages when switching the accounts. 
- Navigate back to the categories page if the user select the article to edit from the category page (Check the existence of history, if present - use .go(-1)
- Update the styles of article list to make it look similar to the other pages in the product.

### CODE REVIEW

File: app/javascript/dashboard/components/widgets/EmptyState.vue [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div class="empty-state py-16 px-1 ml-0 mr-0">
     <h3
       v-if="title"
-      class="text-slate-700 dark:text-slate-200 block text-center w-full text-4xl font-thin"
+      class="text-slate-700 dark:text-slate-200 block text-center w-full text-xl font-thin"
```
Comment (pranavrajs) [inline app/javascript/dashboard/components/widgets/EmptyState.vue:5]: Felt this to be a bit bigger than it should be. 

| Before | After |
| -- | -- |
| Screenshot 2024-03-03 at 7 51 34 PM | Screenshot 2024-03-03 at 7 51 20 PM |

File: app/javascript/dashboard/components/widgets/conversation/contextMenu/menuItem.vue [hunk 2]
```diff
@@ -1,5 +1,9 @@
 <template>
-  <div class="menu" role="button" @click.stop="$emit('click')">
+  <div
```
Comment (pranavrajs) [inline app/javascript/dashboard/components/widgets/conversation/contextMenu/menuItem.vue:2]: The colors here felt a bit light after the Foundation CSS changes. Update it here. 

| Before | After |
| -- | -- |
|  Screenshot 2024-03-03 at 7 53 20 PM | Screenshot 2024-03-03 at 7 52 47 PM |

**Review by iamsivin — COMMENTED** [review]
@pranavrajs Found an issue: the drag functionality is not working and is throwing some errors.
 https://www.loom.com/share/bfd1286ca6914f1982fa3961058b2d32?sid=a31ed994-bbeb-4459-8067-cc5e25cebe34

**Review by iamsivin — APPROVED** [review]
Looks good

**Comment by 01406013368** [comment]
Golry casino

**Pull request merged**